src/server/templates/settings.html

```diff
@@ -2665,10 +2674,18 @@ <h2>{{ __('System') }}</h2>
 	</div>
 	<div class="panel-content">
 		<div class="control-set">
+			{% if is_raspberry_pi %}
```

Comment:
Should there be additional checks here to see if sudo is available?

Reply:
If 'sudo' is not available on the Pi then the user has done something funky with the Pi config, which I don't think we can account for.

Reply:
But the server can be run as a regular user that doesn't have sudo privileges. Wouldn't that make it impossible to run this command?

Reply:
The command works regardless of how the server is run.

Reply:
I should say, assuming that it's run as the 'pi' user on a Raspberry Pi in the standard/default configuration.

Reply:
What's here is better than what came before. We can consider further improvements in another PR.